integration-test/src/test/java/org/apache/gravitino/integration/test/trino/TrinoQueryTestTool.java

```diff
@@ -47,6 +47,12 @@ public static void main(String[] args) throws Exception {
           false,
           "Generate the output file for the test set, the default value is 'false'");
 
+      options.addOption(
+          "test_host",
+          true,
+          "host address for all test services (like gravitino server, trino, hive, postgresql, mysql..., "
+              + "if you services not running on one host. you need to set the arguments like --gravitino_uri=xxx and --trino_uri=xxx), "
+              + "default services address is 127.0.0.1, if --auto is set to 'all', this option is ignored");
```

Comment:
suggestions: 
- Host address for all test services (including .........
- if the services are not running on the same host, set arguments like --gravitino_uri=xxx and --trino_uri=xxx),
- all test services will use 127.0.0.1 as a default host. If --auto is set to 'all', this option is ignored